### ISSUE TITLE
[Umbrella Issue] Migrate prow jobs to community clusters

### ISSUE THREAD

**Issue by rjsadow** [kubernetes/test-infra#29722]
In an ongoing effort to migrate to community-owned resources, SIG K8S Infra and SIG Testing are working to complete the migration of jobs from the Google-owned internal GKE cluster to community-owned clusters.

All jobs in the [Prow Default Cluster](https://prow.k8s.io/?cluster=default) that do not depend on external cloud assets should attempt migrate to `cluster: eks-prow-build-cluster`. 

<h1> What needs to be done? </h1>

<h3> To get started please see <a href='https://github.com/kubernetes/test-infra/blob/master/docs/eks-jobs-migration.md'>eks-jobs-migration</a> for details.</h3>

Fork and check out the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository, then follow the steps below:

1. Find the name of the job you wish to check has a cluster specified or not say `pull-jobset-test-integration-main` from the `"Prow Results"` link below.
2. Edit the file that `pull-jobset-test-integration-main` is defined in from the `"Search Results"` link, in the job definition look for a `cluster:` key, if there isn't one then the job runs in the default cluster, So add one `cluster: eks-prow-build-cluster`. **NOTE**: if you see any entries under `label` that says `gce` skip this job and go to the next time as this is not ready to be moved yet.
3. Save the file, commit the change, create a branch and file a PR
4. Having trouble? Leave a note here in this issue and/or come to #sig-k8s-infra or #sig-testing slack channel to ask for help

**NOTE**: The Google-owned clusters did not require any resource definitions whereas the community-owned clusters do. If your merge is failing the `pull-test-infra-unit-test` job, please add  CPU/Memory requests/limits. Work with the appropriate sig owners to determine the necessary capacity for each job.

Below is a list of repos that currently have jobs in the default cluster. 

**Repos with default cluster jobs found**
- [x] [containerd/containerd](https://github.com/containerd/containerd) | [Prow Results](https://prow.k8s.io/?repo=containerd%2Fcontainerd&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-containerd-build%7Cpull-containerd-node-e2e-1-6%7Cpull-containerd-node-e2e-1-7&i=nope&files=config%2Fjobs%2Fcontainerd%2Fcontainerd) |
- [x] [kubernetes-csi/csi-driver-nfs](https://github.com/kubernetes-csi/csi-driver-nfs) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-csi%2Fcsi-driver-nfs&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-csi-driver-nfs-e2e%7Cpull-csi-driver-nfs-external-e2e%7Cpull-csi-driver-nfs-integration%7Cpull-csi-driver-nfs-sanity%7Cpull-csi-driver-nfs-unit%7Cpull-kubernetes-csi-csi-driver-nfs&i=nope&files=config%2Fjobs%2Fkubernetes-csi%2Fcsi-driver-nfs) | https://github.com/kubernetes/test-infra/pull/30330
- [x] [kubernetes-csi/csi-test](https://github.com/kubernetes-csi/csi-test) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-csi%2Fcsi-test&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubernetes-csi-csi-test&i=nope&files=config%2Fjobs%2Fkubernetes-csi%2Fcsi-test) | https://github.com/kubernetes/test-infra/pull/30330
- [x] [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-csi%2Fexternal-provisioner&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubernetes-csi-external-provisioner-1-24-on-kubernetes-1-24%7Cpull-kubernetes-csi-external-provisioner-1-25-on-kubernetes-1-25%7Cpull-kubernetes-csi-external-provisioner-1-26-on-kubernetes-1-26%7Cpull-kubernetes-csi-external-provisioner-distributed-on-kubernetes-1-26%7Cpull-kubernetes-csi-external-provisioner-unit&i=nope&files=config%2Fjobs%2Fkubernetes-csi%2Fexternal-provisioner) | https://github.com/kubernetes/test-infra/pull/30330
- [x] [kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-csi%2Fexternal-snapshotter&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubernetes-csi-external-snapshotter-1-24-on-kubernetes-1-24%7Cpull-kubernetes-csi-external-snapshotter-1-25-on-kubernetes-1-25%7Cpull-kubernetes-csi-external-snapshotter-1-26-on-kubernetes-1-26%7Cpull-kubernetes-csi-external-snapshotter-unit&i=nope&files=config%2Fjobs%2Fkubernetes-csi%2Fexternal-snapshotter) | https://github.com/kubernetes/test-infra/pull/29812
- [x] [kubernetes-sigs/aws-ebs-csi-driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Faws-ebs-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-aws-ebs-csi-driver-test-e2e-external-eks-windows%7Cpull-aws-ebs-csi-driver-e2e-multi-az%7Cpull-aws-ebs-csi-driver-e2e-single-az%7Cpull-aws-ebs-csi-driver-external-test%7Cpull-aws-ebs-csi-driver-external-test-eks%7Cpull-aws-ebs-csi-driver-test-helm-chart%7Cpull-aws-ebs-csi-driver-unit%7Cpull-aws-ebs-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Faws-ebs-csi-driver) | https://github.com/kubernetes/test-infra/pull/29831
- [x] [kubernetes-sigs/aws-efs-csi-driver](https://github.com/kubernetes-sigs/aws-efs-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Faws-efs-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-aws-efs-csi-driver-e2e%7Cpull-aws-efs-csi-driver-external-test-eks%7Cpull-aws-efs-csi-driver-unit%7Cpull-aws-efs-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Faws-efs-csi-driver) | https://github.com/kubernetes/test-infra/pull/29880
- [x] [kubernetes-sigs/aws-load-balancer-controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Faws-load-balancer-controller&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-aws-load-balancer-controller-e2e-test%7Cpull-aws-load-balancer-controller-lint%7Cpull-aws-load-balancer-controller-unit-test&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Faws-load-balancer-controller) | https://github.com/kubernetes/test-infra/pull/29879
- [x] ~~[kubernetes-sigs/azuredisk-csi-driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fazuredisk-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-azuredisk-csi-driver-e2e-migration-windows%7Cpull-azuredisk-csi-driver-e2e-multi-az%7Cpull-azuredisk-csi-driver-e2e-capz-windows-2022%7Cpull-azuredisk-csi-driver-e2e-capz-windows-2022-hostprocess%7Cpull-azuredisk-csi-driver-integration%7Cpull-azuredisk-csi-driver-e2e-capz%7Cpull-azuredisk-csi-driver-e2e-capz-vmssflex%7Cpull-azuredisk-csi-driver-e2e-capz-windows-2019%7Cpull-azuredisk-csi-driver-e2e-capz-windows-2019-hostprocess%7Cpull-azuredisk-csi-driver-e2e-migration%7Cpull-azuredisk-csi-driver-e2e-single-az%7Cpull-azuredisk-csi-driver-e2e-windows-containerd%7Cpull-azuredisk-csi-driver-external-e2e-single-az%7Cpull-azuredisk-csi-driver-windows-build%7Cpull-azuredisk-csi-driver-sanity%7Cpull-azuredisk-csi-driver-unit%7Cpull-azuredisk-csi-driver-verify%7Cpull-azuredisk-csi-driver-external-e2e-single-az-mainv2%7Cpull-azuredisk-csi-driver-e2e-capz-windows-2019-mainv2&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fazuredisk-csi-driver)~~ | Jobs require azure credentials
- [x] ~~[kubernetes-sigs/azurefile-csi-driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fazurefile-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-azurefile-csi-driver-e2e-capz-windows-2019%7Cpull-azurefile-csi-driver-e2e-capz-windows-2019-hostprocess%7Cpull-azurefile-csi-driver-verify%7Cpull-azurefile-csi-driver-e2e-capz%7Cpull-azurefile-csi-driver-e2e-capz-windows-2022-hostprocess%7Cpull-azurefile-csi-driver-e2e%7Cpull-azurefile-csi-driver-e2e-migration%7Cpull-azurefile-csi-driver-e2e-migration-windows%7Cpull-azurefile-csi-driver-e2e-vmss%7Cpull-azurefile-csi-driver-e2e-windows-containerd%7Cpull-azurefile-csi-driver-external-e2e-nfs%7Cpull-azurefile-csi-driver-external-e2e-smb%7Cpull-azurefile-csi-driver-integration%7Cpull-azurefile-csi-driver-sanity%7Cpull-azurefile-csi-driver-unit%7Cpull-azurefile-csi-driver-windows-build&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fazurefile-csi-driver)~~ | Jobs require azure credentials
- [x] [kubernetes-sigs/cli-utils](https://github.com/kubernetes-sigs/cli-utils) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcli-utils&cluster=default) | [Search Results](https://cs.k8s.io/?q=cli-utils-presubmit-master%7Ccli-utils-presubmit-master-e2e%7Ccli-utils-presubmit-master-stress&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcli-utils) | https://github.com/kubernetes/test-infra/pull/29742
- [x] ~~[kubernetes-sigs/cloud-provider-azure](https://github.com/kubernetes-sigs/cloud-provider-azure) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcloud-provider-azure&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cloud-provider-azure-check%7Cpull-cloud-provider-azure-e2e-capz%7Cpull-cloud-provider-azure-e2e-ccm-capz%7Cpull-cloud-provider-azure-e2e-ccm-vmss-capz%7Cpull-cloud-provider-azure-e2e-ccm-vmss-ip-lb-capz%7Cpull-cloud-provider-azure-e2e-ccm-vmssflex-capz%7Cpull-cloud-provider-azure-unit%7Cpull-cloud-provider-azure-check-1-27%7Cpull-cloud-provider-azure-e2e-capz-1-27%7Cpull-cloud-provider-azure-e2e-ccm-capz-1-27%7Cpull-cloud-provider-azure-e2e-ccm-vmss-capz-1-27%7Cpull-cloud-provider-azure-unit-1-27%7Cpull-cloud-provider-azure-e2e-ccm-vmss-capz-1-26%7Cpull-cloud-provider-azure-check-1-26%7Cpull-cloud-provider-azure-e2e-capz-1-26%7Cpull-cloud-provider-azure-e2e-ccm-capz-1-26%7Cpull-cloud-provider-azure-unit-1-26%7Cpull-cloud-provider-azure-check-1-25%7Cpull-cloud-provider-azure-e2e-capz-1-25%7Cpull-cloud-provider-azure-e2e-ccm-capz-1-25%7Cpull-cloud-provider-azure-e2e-ccm-vmss-capz-1-25%7Cpull-cloud-provider-azure-unit-1-25%7Cpull-cloud-provider-azure-check-1-24%7Cpull-cloud-provider-azure-e2e-capz-1-24%7Cpull-cloud-provider-azure-e2e-ccm-capz-1-24%7Cpull-cloud-provider-azure-e2e-ccm-vmss-capz-1-24%7Cpull-cloud-provider-azure-unit-1-24%7Cpull-cloud-provider-azure-e2e-ccm-ipv6-capz%7Cpull-cloud-provider-azure-e2e-ccm-ipv6-vmss-capz&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcloud-provider-azure)~~ | Jobs require azure credentials
- [x] [kubernetes-sigs/cluster-api](https://github.com/kubernetes-sigs/cluster-api) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-apidiff-main%7Cpull-cluster-api-build-main%7Cpull-cluster-api-e2e-informing-main%7Cpull-cluster-api-e2e-main%7Cpull-cluster-api-test-main%7Cpull-cluster-api-verify-main%7Cpull-cluster-api-apidiff-release-1-4%7Cpull-cluster-api-build-release-1-4%7Cpull-cluster-api-e2e-informing-ipv6-release-1-4%7Cpull-cluster-api-e2e-informing-release-1-4%7Cpull-cluster-api-e2e-release-1-4%7Cpull-cluster-api-test-mink8s-release-1-4%7Cpull-cluster-api-test-release-1-4%7Cpull-cluster-api-verify-release-1-4&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api) | https://github.com/kubernetes-sigs/cluster-api/issues/8689
- [x] [kubernetes-sigs/cluster-api-addon-provider-helm](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-addon-provider-helm&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-addon-provider-helm-apidiff-main%7Cpull-cluster-api-addon-provider-helm-build-main%7Cpull-cluster-api-addon-provider-helm-test-main%7Cpull-cluster-api-addon-provider-helm-test-mink8s-main%7Cpull-cluster-api-addon-provider-helm-verify-main&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-addon-provider-helm) | https://github.com/kubernetes/test-infra/pull/29894
- [x] [kubernetes-sigs/cluster-api-operator](https://github.com/kubernetes-sigs/cluster-api-operator) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-operator&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-operator-e2e-main%7Cpull-cluster-api-operator-apidiff-main%7Cpull-cluster-api-operator-build-main%7Cpull-cluster-api-operator-make-main%7Cpull-cluster-api-operator-test-main%7Cpull-cluster-api-operator-verify-main%7Cpull-cluster-api-operator-e2e&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-operator) | https://github.com/kubernetes-sigs/cluster-api-operator/issues/152
- [x] [kubernetes-sigs/cluster-api-provider-azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-azure&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-provider-azure-apidiff%7Cpull-cluster-api-provider-azure-build%7Cpull-cluster-api-provider-azure-capi-e2e%7Cpull-cluster-api-provider-azure-ci-entrypoint%7Cpull-cluster-api-provider-azure-conformance%7Cpull-cluster-api-provider-azure-conformance-with-ci-artifacts%7Cpull-cluster-api-provider-azure-e2e%7Cpull-cluster-api-provider-azure-e2e-aks%7Cpull-cluster-api-provider-azure-test%7Cpull-cluster-api-provider-azure-verify%7Cpull-cluster-api-provider-azure-windows-containerd-upstream-with-ci-artifacts%7Cpull-cluster-api-provider-azure-apiversion-upgrade%7Cpull-cluster-api-provider-azure-e2e-optional%7Cpull-cluster-api-provider-azure-e2e-workload-upgrade&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-azure) | https://github.com/kubernetes/test-infra/pull/29755
- [x] [kubernetes-sigs/cluster-api-provider-cloudstack](https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-cloudstack&cluster=default) | [Search Results](https://cs.k8s.io/?q=capi-provider-cloudstack-presubmit-build%7Ccapi-provider-cloudstack-presubmit-e2e-smoke-test%7Ccapi-provider-cloudstack-presubmit-unit-test&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-cloudstack) | https://github.com/kubernetes/test-infra/pull/29877#issuecomment-1599344284
- [x] [kubernetes-sigs/cluster-api-provider-digitalocean](https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-digitalocean&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-provider-digitalocean-conformance%7Cpull-cluster-api-provider-digitalocean-e2e%7Cpull-cluster-api-provider-digitalocean-capi-e2e&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-digitalocean) | https://github.com/kubernetes/test-infra/pull/29449
- [x] [kubernetes-sigs/cluster-api-provider-gcp](https://github.com/kubernetes-sigs/cluster-api-provider-gcp) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-gcp&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-provider-gcp-e2e-test%7Cpull-cluster-api-provider-gcp-make%7Cpull-cluster-api-provider-gcp-e2e-test-release-1-3%7Cpull-cluster-api-provider-gcp-make-release-1-3%7Cpull-cluster-api-provider-gcp-e2e-workload-upgrade%7Cpull-cluster-api-provider-gcp-capi-e2e%7Cpull-cluster-api-provider-gcp-conformance&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-gcp) | https://github.com/kubernetes/test-infra/pull/29448 https://github.com/kubernetes/test-infra/pull/29448
- [x] [kubernetes-sigs/cluster-api-provider-ibmcloud](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-ibmcloud&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-provider-ibmcloud-apidiff%7Cpull-cluster-api-provider-ibmcloud-build%7Cpull-cluster-api-provider-ibmcloud-coverage%7Cpull-cluster-api-provider-ibmcloud-make%7Cpull-cluster-api-provider-ibmcloud-smoke-test%7Cpull-cluster-api-provider-ibmcloud-test%7Cpull-cluster-api-provider-ibmcloud-verify%7Cpull-cluster-api-provider-ccm-image&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-ibmcloud) | https://github.com/kubernetes/test-infra/pull/29886
- [x] [kubernetes-sigs/cluster-api-provider-openstack](https://github.com/kubernetes-sigs/cluster-api-provider-openstack) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api-provider-openstack&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cluster-api-provider-openstack-build%7Cpull-cluster-api-provider-openstack-e2e-test%7Cpull-cluster-api-provider-openstack-test%7Cpull-cluster-api-provider-openstack-e2e-full-test&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fcluster-api-provider-openstack) | https://github.com/kubernetes/test-infra/pull/29739
- [x] [kubernetes-sigs/descheduler](https://github.com/kubernetes-sigs/descheduler) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fdescheduler&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-descheduler-test-e2e-k8s-master-1-25%7Cpull-descheduler-test-e2e-k8s-master-1-26%7Cpull-descheduler-test-e2e-k8s-master-1-27%7Cpull-descheduler-unit-test-master-master%7Cpull-descheduler-verify-master%7Cpull-descheduler-verify-build-master&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fdescheduler) | https://github.com/kubernetes/test-infra/pull/29754
- [x] [kubernetes-sigs/gateway-api](https://github.com/kubernetes-sigs/gateway-api) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fgateway-api&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-gateway-api-test%7Cpull-gateway-api-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fgateway-api) | https://github.com/kubernetes/test-infra/pull/29753
- [x] ~~[kubernetes-sigs/gcp-compute-persistent-disk-csi-driver](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fgcp-compute-persistent-disk-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-gcp-compute-persistent-disk-csi-driver-e2e%7Cpull-gcp-compute-persistent-disk-csi-driver-kubernetes-integration%7Cpull-gcp-compute-persistent-disk-csi-driver-sanity%7Cpull-gcp-compute-persistent-disk-csi-driver-unit%7Cpull-gcp-compute-persistent-disk-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fgcp-compute-persistent-disk-csi-driver)~~ | Jobs require GCP
- [x] ~~[kubernetes-sigs/gcp-filestore-csi-driver](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fgcp-filestore-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-gcp-filestore-csi-driver-e2e%7Cpull-gcp-filestore-csi-driver-kubernetes-integration%7Cpull-gcp-filestore-csi-driver-sanity%7Cpull-gcp-filestore-csi-driver-unit%7Cpull-gcp-filestore-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fgcp-filestore-csi-driver)~~ | Jobs require GCP
- [x] [kubernetes-sigs/ibm-powervs-block-csi-driver](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fibm-powervs-block-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-ibm-powervs-block-csi-driver-build%7Cpull-ibm-powervs-block-csi-driver-image-build%7Cpull-ibm-powervs-block-csi-driver-unit%7Cpull-ibm-powervs-block-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fibm-powervs-block-csi-driver) | https://github.com/kubernetes/test-infra/pull/29843
- [x] [kubernetes-sigs/ibm-vpc-block-csi-driver](https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fibm-vpc-block-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-ibm-vpc-block-csi-driver-build%7Cpull-ibm-vpc-block-csi-driver-unit%7Cpull-ibm-vpc-block-csi-driver-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fibm-vpc-block-csi-driver) | https://github.com/kubernetes/test-infra/pull/29844
- [x] [kubernetes-sigs/image-builder](https://github.com/kubernetes-sigs/image-builder) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fimage-builder&cluster=default) | [Search Results](https://cs.k8s.io/?q=json-sort-check%7Cpull-azure-vhds%7Cpull-goss-populate%7Cpull-ova-all%7Cpull-packer-validate&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fimage-builder) | https://github.com/kubernetes/test-infra/pull/29752
- [x] [kubernetes-sigs/ingress-controller-conformance](https://github.com/kubernetes-sigs/ingress-controller-conformance) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fingress-controller-conformance&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-ingress-controller-conformance-boilerplate%7Cpull-ingress-controller-conformance-gherkin%7Cpull-ingress-controller-conformance-build%7Cpull-ingress-controller-conformance-codegen%7Cpull-ingress-controller-conformance-gofmt%7Cpull-ingress-controller-conformance-golint&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fingress-controller-conformance) | https://github.com/kubernetes/test-infra/pull/29750
- [x] [kubernetes-sigs/jobset](https://github.com/kubernetes-sigs/jobset) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fjobset&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-jobset-test-e2e-main-1-24%7Cpull-jobset-test-e2e-main-1-25%7Cpull-jobset-test-e2e-main-1-26%7Cpull-jobset-test-integration-main%7Cpull-jobset-test-unit-main%7Cpull-jobset-verify-main&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fjobset) | https://github.com/kubernetes/test-infra/pull/29756
- [x] [kubernetes-sigs/kernel-module-management](https://github.com/kubernetes-sigs/kernel-module-management) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkernel-module-management&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kernel-module-management-build%7Cpull-kernel-module-management-check-api-changes%7Cpull-kernel-module-management-check-commits-count%7Cpull-kernel-module-management-lint%7Cpull-kernel-module-management-unit-tests&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkernel-module-management) | https://github.com/kubernetes/test-infra/pull/29749
- [x] ~~[kubernetes-sigs/kube-storage-version-migrator](https://github.com/kubernetes-sigs/kube-storage-version-migrator) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkube-storage-version-migrator&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kube-storage-version-migrator-fully-automated-e2e%7Cpull-kube-storage-version-migrator-ha-master%7Cpull-kube-storage-version-migrator-manually-launched-e2e%7Cpull-kube-storage-version-migrator-test%7Cpull-kube-storage-version-migrator-disruptive&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkube-storage-version-migrator)~~
- [x] [kubernetes-sigs/kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkubebuilder&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubebuilder-e2e-k8s-1-25-3%7Cpull-kubebuilder-e2e-k8s-1-26-0%7Cpull-kubebuilder-e2e-k8s-1-27-1%7Cpull-kubebuilder-test&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkubebuilder) | https://github.com/kubernetes/test-infra/pull/29757
- [x] [kubernetes-sigs/kubespray](https://github.com/kubernetes-sigs/kubespray) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkubespray&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubespray-yamllint&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkubespray) | https://github.com/kubernetes/test-infra/pull/29758
- [x] [kubernetes-sigs/kubetest2](https://github.com/kubernetes-sigs/kubetest2) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkubetest2&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubetest2-build%7Cpull-kubetest2-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkubetest2) | https://github.com/kubernetes/test-infra/pull/29760
- [x] [kubernetes-sigs/kustomize](https://github.com/kubernetes-sigs/kustomize) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkustomize&cluster=default) | [Search Results](https://cs.k8s.io/?q=kustomize-presubmit-master&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkustomize) | https://github.com/kubernetes/test-infra/pull/29733
- [x] [kubernetes-sigs/kwok](https://github.com/kubernetes-sigs/kwok) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fkwok&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kwok-build-main%7Cpull-kwok-e2e-test-main%7Cpull-kwok-unit-test-main%7Cpull-kwok-verify-main&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fkwok) | https://github.com/kubernetes/test-infra/pull/29731
- [x] [kubernetes-sigs/node-feature-discovery](https://github.com/kubernetes-sigs/node-feature-discovery) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fnode-feature-discovery&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-node-feature-discovery-build-image-cross-generic%7Cpull-node-feature-discovery-build-image-generic%7Cpull-node-feature-discovery-build-master%7Cpull-node-feature-discovery-verify-master%7Cpull-node-feature-discovery-build-gh-pages-generic%7Cpull-node-feature-discovery-verify-docs-master%7Cpostsubmit-node-feature-discovery-e2e-test%7Cpostsubmit-node-feature-discovery-verify-master&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fnode-feature-discovery) | https://github.com/kubernetes/test-infra/pull/29761
- [x] [kubernetes-sigs/prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fprometheus-adapter&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-prometheus-adapter-test-e2e%7Cpull-prometheus-adapter-verify%7Cpull-prometheus-adapter-test&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fprometheus-adapter) | https://github.com/kubernetes/test-infra/pull/29766
- [x] [kubernetes-sigs/promo-tools](https://github.com/kubernetes-sigs/promo-tools) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fpromo-tools&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cip-auditor-e2e%7Cpull-cip-e2e&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fpromo-tools) | https://github.com/kubernetes/test-infra/pull/29436
- [x] [kubernetes-sigs/security-profiles-operator](https://github.com/kubernetes-sigs/security-profiles-operator) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fsecurity-profiles-operator&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-security-profiles-operator-build%7Cpull-security-profiles-operator-build-image%7Cpull-security-profiles-operator-test-e2e%7Cpull-security-profiles-operator-test-unit%7Cpull-security-profiles-operator-verify&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fsecurity-profiles-operator) | https://github.com/kubernetes/test-infra/pull/29768
- [x] [kubernetes-sigs/vsphere-csi-driver](https://github.com/kubernetes-sigs/vsphere-csi-driver) | [Prow Results](https://prow.k8s.io/?repo=kubernetes-sigs%2Fvsphere-csi-driver&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-vsphere-csi-driver-unit-test%7Cpull-vsphere-csi-driver-verify-golangci-lint%7Cpull-vsphere-csi-driver-verify-markdown%7Cpull-vsphere-csi-driver-verify-staticcheck%7Cpull-vsphere-csi-driver-verify-vet%7Cpull-vsphere-csi-driver-verify-fmt%7Cpull-vsphere-csi-driver-verify-shell%7Cpost-vsphere-csi-driver-deploy%7Cpull-vsphere-csi-driver-build&i=nope&files=config%2Fjobs%2Fkubernetes-sigs%2Fvsphere-csi-driver) | https://github.com/kubernetes/test-infra/pull/29849
- [x] [kubernetes/cloud-provider-aws](https://github.com/kubernetes/cloud-provider-aws) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fcloud-provider-aws&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cloud-provider-aws-check%7Cpull-cloud-provider-aws-e2e%7Cpull-cloud-provider-aws-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fcloud-provider-aws) | https://github.com/kubernetes/test-infra/pull/29850
- [x] [kubernetes/cloud-provider-gcp](https://github.com/kubernetes/cloud-provider-gcp) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fcloud-provider-gcp&cluster=default) | [Search Results](https://cs.k8s.io/?q=cloud-provider-gcp-e2e-full%7Ccloud-provider-gcp-tests%7Ccloud-provider-gcp-verify-all%7Ccloud-provider-gcp-verify-up-to-date%7Cpull-cloud-provider-gcp-scenario-kops-simple%7Ccloud-provider-gcp-verify-crds&i=nope&files=config%2Fjobs%2Fkubernetes%2Fcloud-provider-gcp) | https://github.com/kubernetes/test-infra/pull/29852
- [x] [kubernetes/cloud-provider-openstack](https://github.com/kubernetes/cloud-provider-openstack) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fcloud-provider-openstack&cluster=default) | [Search Results](https://cs.k8s.io/?q=openstack-cloud-controller-manager-e2e-test%7Cpull-cloud-provider-openstack-check%7Cpull-cloud-provider-openstack-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fcloud-provider-openstack) | https://github.com/kubernetes/test-infra/pull/29853
- [x] [kubernetes/cloud-provider-vsphere](https://github.com/kubernetes/cloud-provider-vsphere) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fcloud-provider-vsphere&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-cloud-provider-vsphere-build%7Cpull-cloud-provider-vsphere-coverage%7Cpull-cloud-provider-vsphere-e2e-test%7Cpull-cloud-provider-vsphere-integration-test%7Cpull-cloud-provider-vsphere-unit-test%7Cpull-cloud-provider-vsphere-verify-fmt%7Cpull-cloud-provider-vsphere-verify-lint%7Cpull-cloud-provider-vsphere-verify-staticcheck%7Cpull-cloud-provider-vsphere-verify-vet&i=nope&files=config%2Fjobs%2Fkubernetes%2Fcloud-provider-vsphere) | https://github.com/kubernetes/test-infra/pull/29854
- [x] [kubernetes/community](https://github.com/kubernetes/community) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fcommunity&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-community-tempelis-check%7Cpull-community-verify&i=nope&files=config%2Fjobs%2Fkubernetes%2Fcommunity) | https://github.com/kubernetes/test-infra/pull/29737
- [x] [kubernetes/dns](https://github.com/kubernetes/dns) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fdns&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kubernetes-dns-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fdns) | https://github.com/kubernetes/test-infra/pull/30384
- [x] [kubernetes/enhancements](https://github.com/kubernetes/enhancements) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fenhancements&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-enhancements-test%7Cpull-enhancements-verify&i=nope&files=config%2Fjobs%2Fkubernetes%2Fenhancements) | https://github.com/kubernetes/test-infra/pull/29737
- [x] [kubernetes/k8s.io](https://github.com/kubernetes/k8s.io) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fk8s.io&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-k8sio-groups-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fk8s.io) | https://github.com/kubernetes/test-infra/pull/30379
- [x] [kubernetes/kops](https://github.com/kubernetes/kops) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fkops&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-kops-build%7Cpull-kops-e2e-k8s-aws-calico%7Cpull-kops-e2e-k8s-gce-cilium%7Cpull-kops-e2e-k8s-gce-ipalias%7Cpull-kops-test%7Cpull-kops-verify-boilerplate%7Cpull-kops-verify-generated%7Cpull-kops-verify-gofmt%7Cpull-kops-verify-golangci-lint%7Cpull-kops-verify-gomod%7Cpull-kops-verify-govet%7Cpull-kops-e2e-aws-karpenter%7Cpull-kops-e2e-aws-upgrade-126-ko126-to-klatest-kolatest-many-addons%7Cpull-kops-e2e-aws-upgrade-k123-ko125-to-k124-kolatest-karpenter%7Cpull-kops-verify-terraform%7Ckops-postsubmit%7Cpull-kops-aws-distro-rhel8%7Cpull-kops-aws-distro-rocky8%7Cpull-kops-aws-distro-rhel9&i=nope&files=config%2Fjobs%2Fkubernetes%2Fkops) | https://github.com/kubernetes/kops/issues/16637
- [x] [kubernetes/minikube](https://github.com/kubernetes/minikube) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fminikube&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-minikube-build&i=nope&files=config%2Fjobs%2Fkubernetes%2Fminikube) | https://github.com/kubernetes/test-infra/pull/29724
- [x] [kubernetes/node-problem-detector](https://github.com/kubernetes/node-problem-detector) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fnode-problem-detector&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-npd-build%7Cpull-npd-e2e-kubernetes-gce-gci%7Cpull-npd-e2e-kubernetes-gce-gci-custom-flags%7Cpull-npd-e2e-kubernetes-gce-ubuntu%7Cpull-npd-e2e-kubernetes-gce-ubuntu-custom-flags%7Cpull-npd-e2e-node%7Cpull-npd-e2e-test%7Cpull-npd-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fnode-problem-detector) | https://github.com/kubernetes/test-infra/pull/29751
- [x] [kubernetes/org](https://github.com/kubernetes/org) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Forg&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-org-test-all%7Cpull-org-verify-all&i=nope&files=config%2Fjobs%2Fkubernetes%2Forg) | https://github.com/kubernetes/test-infra/pull/29737

**Comment by ameukam** [comment]
For kops, we have a separate issue: https://github.com/kubernetes/k8s.io/issues/5127

**Comment by dims** [comment]
# What needs to be done?

- Find the name of the job you wish to check has a cluster specified or not say `pull-jobset-test-integration-main` from the `"Search Results"` link above

- Find the yaml file in `kubernetes/test-infra` using our search:
  - https://cs.k8s.io/?q=pull-jobset-test-integration-main&i=nope&files=&excludeFiles=&repos=

- Fork and check out the test-infra repository

- Edit the file in step 2, in the job definition look for a `cluster:` key, if there isn't one then the job runs in the `default` cluster, So add one `cluster: eks-prow-build-cluster`

NOTE: if you see any entries under `label` that says `gce` skip this job and go to the next time as this is not ready to be moved yet.

- Save the file, commit the change, create a branch and file a PR

- Having trouble? Leave a note here in this issue and/or come to #sig-k8s-infra or #sig-testing slack channel to ask for help

**Comment by rayandas** [comment]
@dims Can I pick up the `cluster-api` jobs? I will migrate as much as I can.

**Comment by ameukam** [comment]
> @dims Can I pick up the `cluster-api` jobs? I will migrate as much as I can.

@rayandas @rjsadow This is an ongoing [effort](https://github.com/kubernetes-sigs/cluster-api/issues/8689) about `cluster-api`. Please coordinate with cluster-api maintainers

**Comment by rjsadow** [comment]
@dims @ameukam I think the `pull-test-infra-unit-test` job requiring resoure limits/requests for the community clusters is going to be a stumbling block for a lot of these migrations. Is there any existing documentation or resources that we can empower contributers to reference to for setting initial values then iterating? Or should we expect the sig maintainters and leads to help provide input on a job-by-job basis? 

Example in https://github.com/kubernetes/test-infra/pull/29724 where I WAG'd .5CPU and 2GB and asked the cluster lifecycle leads to review.

**Comment by furkatgofurov7** [comment]
> # What needs to be done?
> Fork and check out the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository, then follow the steps below:
> 
> 1. Find the name of the job you wish to check has a cluster specified or not say `pull-jobset-test-integration-main` from the `"Search Results"` link above
> 2. Find the yaml file in `kubernetes/test-infra` using our search:
>    
>    * https://cs.k8s.io/?q=pull-jobset-test-integration-main&i=nope&files=&excludeFiles=&repos=
> 3. Edit the file in step 2, in the job definition look for a `cluster:` key, if there isn't one then the job runs in the `default` cluster, So add one `cluster: eks-prow-build-cluster`. NOTE: if you see any entries under `label` that says `gce` skip this job and go to the next time as this is not ready to be moved yet.
> 4. Save the file, commit the change, create a branch and file a PR
> 5. Having trouble? Leave a note here in this issue and/or come to #sig-k8s-infra or #sig-testing slack channel to ask for help

@dims hi, thanks for sharing the steps. Also I noticed PR's moving to new clusters with just specifying the new cluster name, there are others specifying resource quotas (CPU/memory) while migrating. Is the latter requirement or generally maybe recommended while just specifying the name is also enough?

**Comment by rjsadow** [comment]
> Is the latter requirement or generally maybe recommended while just specifying the name is also enough?

The Google-owned clusters did not require any resource definitions whereas the community-owned clusters do. If a job already has resource quotas (both requests and limits) then just the name is enough. If the job is missing any resource quotas then those will need to be added or else the `pull-test-infra-unit-test` check will fail.

**Comment by Vyom-Yadav** [comment]
I will be working on [kubernetes/node-problem-detector](https://github.com/kubernetes/node-problem-detector) | [Prow Results](https://prow.k8s.io/?repo=kubernetes%2Fnode-problem-detector&cluster=default) | [Search Results](https://cs.k8s.io/?q=pull-npd-build%7Cpull-npd-e2e-kubernetes-gce-gci%7Cpull-npd-e2e-kubernetes-gce-gci-custom-flags%7Cpull-npd-e2e-kubernetes-gce-ubuntu%7Cpull-npd-e2e-kubernetes-gce-ubuntu-custom-flags%7Cpull-npd-e2e-node%7Cpull-npd-e2e-test%7Cpull-npd-test&i=nope&files=config%2Fjobs%2Fkubernetes%2Fnode-problem-detector)

**Comment by ameukam** [comment]
@rjsadow All jobs running against `github.com/kubernetes-sigs/cluster-api-provider-azure` using prow presets `preset-azure-cred-*` are not eligible to migrate AFAIK. There are using specific creds currently stored in a Google-owned GCP project. 
we can try to sync to creds to the community infra before we do the migration.

**Comment by ameukam** [comment]
@cpanato can you tag this issue in the PRs related to the jobs migration ? just want to link issues and PRs easily.

**Comment by cpanato** [comment]
> @cpanato can you tag this issue in the PRs related to the jobs migration ? just want to link issues and PRs easily.

@ameukam added the PR above

**Comment by pohly** [comment]
Regarding resource requirements see also https://github.com/kubernetes/test-infra/issues/28800

**Comment by AverageMarcus** [comment]
Just an FYI:

We've had to revert one of the jobs used by `image-builder` that relies on a secret that isn't migrated to the new `eks-prow-build-cluster`.

See [PR here](https://github.com/kubernetes/test-infra/pull/29896), related [issue here](https://github.com/kubernetes-sigs/image-builder/issues/1206) and [Slack discussion here](https://kubernetes.slack.com/archives/C09QZ4DQB/p1687357644222779).

**Comment by Jont828** [comment]
@iftachk It seems like the presubmit jobs for Cluster API Add-on Provider Helm aren't working. I'm getting an error that the pods failed to start on the PR jobs.

**Comment by rjsadow** [comment]
> How large are the nodes in the EKS cluster, i.e. what is the maximum resource requirement that a job can request?

16 vCPUs, 128 GB RAM, 300 GB NVMe SSD per https://github.com/kubernetes/test-infra/blob/master/docs/eks-jobs-migration.md?plain=1#L21

**Comment by rjsadow** [comment]
Here's a list of outstanding PRs that are waiting to be reviewed:

- https://github.com/kubernetes/test-infra/pull/29731
- https://github.com/kubernetes/test-infra/pull/29831
- https://github.com/kubernetes/test-infra/pull/29841
- https://github.com/kubernetes/test-infra/pull/29842
- https://github.com/kubernetes/test-infra/pull/29845
- https://github.com/kubernetes/test-infra/pull/30025
- https://github.com/kubernetes/test-infra/pull/30035
- https://github.com/kubernetes/test-infra/pull/30103

**Comment by rjsadow** [comment]
For folks still interested in helping with this migration, I've added a new tool to `k/test-infra` to help identify repository status and which jobs are eligible to be transitioned to the community clusters from the GKE cluster. If interested, please take a look at these few gists to see how the tool works. We are getting towards the end of jobs that are currently eligible for transition, so we'll need to discuss how to handle follow on efforts (like https://github.com/kubernetes/test-infra/issues/30277) soon. 

https://gist.github.com/rjsadow/c5d1695c73842e53449a772f00b00116
https://gist.github.com/rjsadow/d6cfe557f6ef3fa1cd2838053e3bb1fd
https://gist.github.com/rjsadow/83c50e252fe0534e03d5aa2eb72503eb

Relevant PRs for the tool and additional info:
https://github.com/kubernetes/test-infra/pull/30442
https://github.com/kubernetes/test-infra/pull/30467
https://kubernetes.slack.com/archives/CCK68P2Q2/p1692149062215619

**Comment by BenTheElder** [comment]
xref: https://github.com/kubernetes/test-infra/issues/32432

**Comment by BenTheElder** [comment]
We're also keeping this list up to date: https://github.com/kubernetes/test-infra/blob/master/docs/job-migration-todo.md

**Comment by BenTheElder** [comment]
filed a few warnings:
https://github.com/kubernetes/kops/issues/16637
https://github.com/kubernetes-sigs/node-feature-discovery/issues/1747
https://github.com/kubernetes/node-problem-detector/issues/920

+ messages in slack, in addition to the kubernetes-dev emails

**Comment by BenTheElder** [comment]
NPD is actually done now. We're getting really close. We have DO spending info now and sorted out credentials, @upodroid has been migrating those jobs. I migrated the slack infra and some more of the test-infra trusted jobs. kops jobs are being migrated with DO as well.

**Comment by BenTheElder** [comment]
secrets-store-csi-driver seems like a new wrinkle https://kubernetes.slack.com/archives/C09QZ4DQB/p1722024643376909

we know that vsphere jobs are blocked on resource availability and may not making it https://github.com/kubernetes/k8s.io/issues/6877

azure jobs are in flight and making steady progress now

most other jobs are done, leaving a handful of trusted project automation jobs in #32432 , and some scalability related jobs

we may have missed others and should do another pass through these to identify the categories

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/test-infra/pull/33127 makes it a little more obvious what grouping remaining jobs are in based on the secrets

Since filed https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/1583 and https://github.com/kubernetes/test-infra/issues/33128 but there are more:

In addition to the vsphere + azure e2e jobs we also have:

- manual scale presubmits https://github.com/kubernetes/test-infra/issues/33128
- assorted janitor jobs (aws, two for vsphere, couple more for scale / gce) (https://github.com/kubernetes/test-infra/issues/33129)
- trusted cluster prow self push / deploy (so ... we'll shut those down just ahead of migration) #32432 
- prow auth update jobs (multiple) #32432 
- some infra / testing image pushing (WIP) #32432 
- testgrid config upload (WIP) #32432 
- secrets-store-csi https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/1583

**Comment by BenTheElder** [comment]
Closing this in favor of https://github.com/kubernetes/test-infra/issues/33226 and https://github.com/kubernetes/test-infra/issues/32432

Announcement at https://groups.google.com/a/kubernetes.io/g/dev/c/qzNYpcN5la4
